data/uhc_pack/functions/pre_generation/stage/generate.mcfunction

```diff
@@ -0,0 +1,19 @@
+scoreboard players set X uhcPG 0
+scoreboard players set Z uhcPG 0
+scoreboard players set Done uhcPG 0
+scoreboard players set Total uhcPG 0
+scoreboard players set Direction uhcPG 1
+
+scoreboard players operation Total uhcPG += UHC uhcBSize
```

Comment:
So it seems like currently the nether generates the same size as the overworld, however we only need to generate 1/8 of the border size (or 1/16 of the chunks).

Reply:
I thought the border size was consistent across dimensions? It's just the center location that changes
I'll have another look but yeah, that was the impression I got from the tests I did

Reply:
Yep, that's the case. The border is only translated, not shrunk :)

Reply:
I'll fix this up post merge. Players are incredibly unlikely to wander out of chunks which translate to overworld chunks, regardless of where the border is. Pre-generation can take quite some time and I think the tradeoff should preference a shorter pre-generation